src/pages/NewChatPage.js

```diff
@@ -258,6 +262,7 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, translate, i
                             isLoadingNewOptions={isSearchingForReports}
                         />
                     </View>
+                    {isSmallScreenWidth && <OfflineIndicator style={[]} />}
```

Comment:
Curious what the purpose of setting the style to `{[]}` is?

Reply:
`style` props of `OfflineIndicator` is an array and it doesn't have default props. Alternatively, we can change the default value of `style` props of the component to `true`

Reply:
@narefyev91

Please let me know your thoughts on https://github.com/Expensify/App/pull/30491#discussion_r1376263589

Reply:
yes let's have a default one

Reply:
@MariaHCD 

Default value added